samples/msal-node-samples/auth-code-distributed-cache/src/AxiosHelper.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import axios from "axios";
+
+class AxiosHelper {
+
+    /**
+     * Makes an Authorization "Bearer" request with the given accessToken to the given endpoint.
+     * @param endpoint
+     * @param accessToken
+     */
+    static async callDownstreamApi(endpoint: string, accessToken?: string, params?: Record<string, string>): Promise<any> {
+        console.log(`Request to ${endpoint} made at: ${new Date().toString()}`);
```

Comment:
Does .toString() actually make a difference here?

Reply:
yes, it's object => string, but yeah, template literal is already doing the same. good catch, removed